scripts/check_ingestion_sequences.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+"""
+Script to facet on the ingesiton sequence number for a given provider
+to help detect when data that should have been deleted from the index remains.
+
+Requires that the ES index url exist in akara.ini, like:
+
+[Elasticsearch]
+Index=http://search-prod1:9200/dpla_alias
+
+Usage:
+    $ python check_ingestion_sequences.py path_to_profile
+"""
+
+import sys
+import argparse
+from amara.thirdparty import json
+import ConfigParser
+import requests
+
+
+def define_arguments():
+    """Defines command line arguments for the current script"""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("profile_path",
+                        help="The path to the profile to be processed")
+
+    return parser
+
+
+def get_provider_id(profile_path):
+    with open(profile_path, "r") as f:
+        try:
+            profile = json.load(f)
+            return profile['contributor']['@id']
+        except Exception, err:
+            print "Error, could not load profile in %s: %s" % (__name__, err)
+            return None
+
+
+def get_elasticsearch_url():
+    config = ConfigParser.ConfigParser(
+        {"LogLevel": "INFO", "SyncQAViews": "True"}
```

Comment:
Cut and paste code. I didn't spend the time to figure out how this config stuff works.